plugin/pkg/admission/gc/gc_admission.go

```diff
@@ -258,7 +258,7 @@ func newBlockingOwnerDeletionRefs(newObj, oldObj runtime.Object) []metav1.OwnerR
 			ret = append(ret, ref)
 			continue
 		}
-		wasNotBlocking := oldRef.BlockOwnerDeletion == nil || *oldRef.BlockOwnerDeletion == false
```

Comment:
prefer the original code block

Reply:
I agree.  Is there a way to fix staticcheck?

Reply:
yes